src/lib.rs

```diff
@@ -354,20 +325,20 @@ macro_rules! prop_allof {
     };
 }
 
-/// Executes a sequence of commands and returns those that were executed.
+/// Executes a sequence of commands and returns those executed.
```

Comment:
Feel free to resolve without applying but it sounds more natural:
```suggestion
/// Executes a sequence of commands and returns the executed ones.
```